### PR TITLE
Task: 03-date-tasks

### PR DESCRIPTION
**Pull request by verenichyi**
Implement `parseDataFromRfc2822`, `parseDataFromIso8601`, `isLeapYear`, `timeSpanToString`, `angleBetweenClockHands` functions.

### CODE REVIEW

File: js/practice/task/03-date-tasks.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
  *    'Sun, 17 May 1998 03:00:00 GMT+01' => Date()
  */
 function parseDataFromRfc2822(value) {
-   throw new Error('Not implemented');
+	return Date.parse(value);
```
Comment (echonok) [inline js/practice/task/03-date-tasks.js]: It does fit tests, but the type of the result is not Date. Please, function must return Date type

File: js/practice/task/03-date-tasks.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ function parseDataFromRfc2822(value) {
  *    '2016-01-19T08:07:37Z' => Date()
  */
 function parseDataFromIso8601(value) {
-   throw new Error('Not implemented');
+	return Date.parse(value);
```
Comment (echonok) [inline js/practice/task/03-date-tasks.js]: Same here

File: js/practice/task/03-date-tasks.js [hunk 3]
```diff
@@ -56,7 +56,8 @@ function parseDataFromIso8601(value) {
  *    Date(2015,1,1)    => false
  */
 function isLeapYear(date) {
-   throw new Error('Not implemented');
+	const year = date.getFullYear();
+	return ((year % 4 === 0) && (year % 100 !== 0)) || (year % 400 === 0);
```
Comment (echonok) [inline js/practice/task/03-date-tasks.js:60]: Very good

**Review by echonok — CHANGES_REQUESTED** [review]

File: js/practice/task/03-date-tasks.js (hunk 1 above)
Comment (verenichyi) [reply js/practice/task/03-date-tasks.js]: Fixed

File: js/practice/task/03-date-tasks.js (hunk 2 above)
Comment (verenichyi) [reply js/practice/task/03-date-tasks.js]: Fixed

File: js/practice/task/03-date-tasks.js (hunk 1 above)
Comment (echonok) [reply js/practice/task/03-date-tasks.js]: What did you fix? :D

Comment (verenichyi) [reply js/practice/task/03-date-tasks.js]: Oh, I'm so sorry!
Forgot to push the commit.

**Review by echonok — APPROVED** [review]

**Pull request merged**